### PR TITLE
adds storage, user and data management pages

### PR DESCRIPTION
**Pull request by ykethan**
#### Description of changes:

Console pages:
User management
file storage
Data manager

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by josefaidt — DISMISSED** [review]

File: src/pages/[platform]/build-a-backend/storage/manage-with-amplify-console/index.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Access File storage
+
+After you've deployed your **Storage** resource, you can access the manager on Amplify Console.
+
+1. Log in to the [Amplify console](https://console.aws.amazon.com/amplify/home) and choose your app.
+2. Select the branch you would like to access.
+3. Select **Storage** from the left navigation bar.
+
+<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
```
Comment (jacoblogan) [inline src/pages/[platform]/build-a-backend/storage/manage-with-amplify-console/index.mdx:45]: please use the `<Video>` component

File: src/pages/[platform]/build-a-backend/data/manage-with-amplify-console/index.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+## Access Data manager
+
+After you've deployed your **Data** resource, you can access the manager on Amplify Console.
+
+1. Log in to the [Amplify console](https://console.aws.amazon.com/amplify/home) and choose your app.
+2. Select the branch you would like to access.
+3. Select **Data** from the left navigation bar.
+4. Then, select **Data manager**.
+
+<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
```
Comment (jacoblogan) [inline src/pages/[platform]/build-a-backend/data/manage-with-amplify-console/index.mdx:46]: Please use the `<Video>` component here

File: src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+## Access User management
+
+After you've deployed your **Auth** resource, you can access the manager on Amplify Console.
+
+1. Log in to the [Amplify console](https://console.aws.amazon.com/amplify/home) and choose your app.
+2. Select the branch you would like to access.
+3. Select **Authentication** from the left navigation bar.
+4. Then, select **User management**.
+
+<video autoPlay={true} muted={true} loop={true} width="100%" playsInline={true}>
```
Comment (jacoblogan) [inline src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx:46]: A wrapper for this video element has been made for docs use `src/components/Video` please use the `<Video>` component here

Comment (ykethan) [reply src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx]: updated to `<Video/>`

**Review by jacoblogan — DISMISSED** [review]

**Review by josefaidt — DISMISSED** [review]

File: src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  };
+}
+
+The **User management** page in the Amplify Console provides a user-friendly interface for managing your application's users. You can create and manage users and groups, edit user attributes, and suspend users. 
+
+If you have not yet created a **Auth** resource, visit the [Auth setup guide](/[platform]/build-a-backend/auth/set-up-auth/)
+
+## Access User management
+
+After you've deployed your **Auth** resource, you can access the manager on Amplify Console.
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx:39]:
```suggestion
After you've deployed your **Auth** resource, you can access the manager in the Amplify console.
```

File: src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+The **User management** page in the Amplify Console provides a user-friendly interface for managing your application's users. You can create and manage users and groups, edit user attributes, and suspend users. 
+
+If you have not yet created a **Auth** resource, visit the [Auth setup guide](/[platform]/build-a-backend/auth/set-up-auth/)
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx:35]:
```suggestion
If you have not yet created an **auth** resource, visit the [Auth setup guide](/[platform]/build-a-backend/auth/set-up-auth/)
```

File: src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx [hunk 6]
```diff
@@ -0,0 +1,81 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Manage users with Amplify console',
+  description:
+    'Manage applications Cognito users and groups with Amplify console',
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx:6]:
```suggestion
    'Manage application's Cognito users and groups with the Amplify console',
```

File: src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+The **User management** page in the Amplify Console provides a user-friendly interface for managing your application's users. You can create and manage users and groups, edit user attributes, and suspend users.
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/auth/manage-users/with-amplify-console/index.mdx:33]:
```suggestion
The **User management** page in the Amplify console provides a user-friendly interface for managing your application's users. You can create and manage users and groups, edit user attributes, and suspend users. 
```

File: src/pages/[platform]/build-a-backend/data/manage-with-amplify-console/index.mdx [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+The **Data manager** page in the Amplify Console offers a user-friendly interface for managing the backend GraphQL API data of an application. It enables real-time creation and updates of application data, eliminating the need to build separate admin views.
+
+If you have not yet created a **Data** resource, visit the [Data setup guide](/[platform]/build-a-backend/data/set-up-data/)
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/data/manage-with-amplify-console/index.mdx:35]:
```suggestion
If you have not yet created a data resource, visit the [Data setup guide](/[platform]/build-a-backend/data/set-up-data/)
```

File: src/pages/[platform]/build-a-backend/data/manage-with-amplify-console/index.mdx [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+## Access Data manager
+
+After you've deployed your **Data** resource, you can access the manager on Amplify Console.
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/data/manage-with-amplify-console/index.mdx:39]:
```suggestion
After you've deployed your data resource, you can access the manager on Amplify console.
```

File: src/pages/[platform]/build-a-backend/data/manage-with-amplify-console/index.mdx [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+3. Select **Data** from the left navigation bar.
+4. Then, select **Data manager**.
+
+<Video src="/images/gen2/manage/data-manager.mp4"/>
+
+### To create a record
+
+1. On the **Data manager** page, select a table from the **Select table** dropdown. For this example, we are using a *Todo* table.
+2. Select **Create Todo**.
+3. In the **Add Todo** pane, specify your custom values for the fields in the table. For example, enter *my first todo* for the *Content* field and toggle the *Is done* field, 4. Then select **Submit**.
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/data/manage-with-amplify-console/index.mdx:52]:
```suggestion
3. In the **Add Todo** pane, specify your custom values for the fields in the table. For example, enter *my first todo* for the *Content* field and toggle the *Is done* field

4. Select **Submit**.
```

File: src/pages/[platform]/build-a-backend/storage/manage-with-amplify-console/index.mdx [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+The **File storage** page in the Amplify Console provides a user-friendly interface for managing your application's backend file storage. It allows for efficient testing and management of your files.
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/storage/manage-with-amplify-console/index.mdx:33]:
```suggestion
The **File storage** page in the Amplify console provides a user-friendly interface for managing your application's backend file storage. It allows for efficient testing and management of your files.
```

File: src/pages/[platform]/build-a-backend/storage/manage-with-amplify-console/index.mdx [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+
+If you have not yet created a **Storage** resource, visit the [Storage setup guide](/[platform]/build-a-backend/storage/set-up-storage/)
```
Comment (aspittel) [inline src/pages/[platform]/build-a-backend/storage/manage-with-amplify-console/index.mdx:35]:
```suggestion
If you have not yet created a **storage** resource, visit the [Storage setup guide](/[platform]/build-a-backend/storage/set-up-storage/)
```

**Review by aspittel — CHANGES_REQUESTED** [review]

**Review by aspittel — APPROVED** [review]

**Review by josefaidt — APPROVED** [review]

**Pull request merged**